### ISSUE TITLE
Add paging, numbers and links to the top of the page.

### ISSUE THREAD

**Issue by phobiadhs** [digitalocean/netbox#871]
Example: When looking at a multi page list of objects, IP's for example (/ipam/ip-addresses/), the paging links are at the bottom of the page. This causes a break in workflow in certain cases, though admittedly a small one. 

I think a simple fix to this is just to add paging to the top of each page so that there are paging options both at the top and the bottom.

**Comment by Armadill0** [comment]
Full ack. I also stumbled about this several times but forgot to report it. :smiley:

**Comment by chagara** [comment]
same here. 1+

**Comment by jeremystretch** [comment]
Where would you put it? A mock-up would be helpful.

**Comment by jeremystretch** [comment]
Closing due to lack of activity.

**Comment by phobiadhs** [comment]
I just noticed that I left this hanging. Sorry about that. 
I would like to request that it be reopened if possible.

To answer your question as to the location, in the same location that I referenced above, there is sufficient space above the description/interface fields to put in a paging function like there is at the bottom of the page. I am happy to provide an image for reference if needed. Just let me know.
